src/components/Header.js

```diff
@@ -8,10 +8,12 @@ import Container from './Container';
 
 const Wrapper = styled.header`
   padding: 25px 0;
+  position: relative;
+  border-bottom: 1px solid #bcd0e5;
 
-  @media screen and (max-width: 800px) {
-    position: relative;
-    border-bottom: 1px solid #bcd0e5;
+  @media screen and (min-width: 800px) {
+    position: static;
```

Comment:
You can leave `position: relative` as it doesn't break anything

Reply:
Done